server/task/src/main/scala/org/apache/james/task/eventsourcing/DecisionProjection.scala

```diff
@@ -0,0 +1,25 @@
+package org.apache.james.task.eventsourcing
+
+import org.apache.james.eventsourcing.eventstore.History
+import org.apache.james.task.TaskManager.Status
+
+case class DecisionProjection(status: Status)
+
+object DecisionProjection {
+  def load(history: History) = {
```

Comment:
The projection should not receive the history but events IMO. Otherwise it would be the only projection of the system to not be event-based